views/cypress/tests/manage-schema.spec.js

```diff
@@ -0,0 +1,129 @@
+                );
+            }
+        });
+    });
+
+    /**
+     * Tests
+     */
+    describe('Main Item Class creation and editing', () => {
+        it('can create a new item class', function () {
+            cy.addClassToRoot(
+                selectors.root,
+                selectors.itemClassForm,
+                className,
+                selectors.editClassLabelUrl,
+                selectors.treeRenderUrl,
+                selectors.addSubClassUrl
+            );
+        });
+
+        it('can edit and add new property for the item class', function () {
+            cy.addPropertyToClass(
+                className,
+                selectors.editClass,
+                selectors.classOptions,
+                newPropertyName,
+                selectors.propertyEdit,
+                selectors.editClassUrl
+            );
+        });
+    });
+
+    describe('Child Item Class', () => {
+        it('create a new child item class', function () {
+            cy.intercept('POST', `**/${ selectors.editClassLabelUrl }`).as('editClassLabel');
+            cy.addClass(
+                selectors.itemClassForm,
+                selectors.treeRenderUrl,
+                selectors.addSubClassUrl
+            );
+            cy.renameSelectedClass(selectors.itemClassForm, childClassName);
+        });
+
+        it('child item class inherits parent property', function() {
+            cy.intercept('POST', `**/${ selectors.editClassUrl }`).as('editClass');
+            cy.getSettled(selectors.editClass).click();
+            cy.wait('@editClass');
+            cy.getSettled(selectors.classOptions)
+              .contains('.property-block', newPropertyName)
+              .contains('.property-heading-toolbar', className)
+              .within(() => {
+                cy.get('.icon-edit').click();
+              });
+            cy.get('.property-edit-container [data-testid="Label"]').should('have.value', newPropertyName);
+        });
+    });
+
+    describe('Child Item', () => {
+        it('create a new child item', function () {
+            cy.selectNode(selectors.root, selectors.itemClassForm, className);
+            cy.addNode(selectors.itemForm, selectors.addItem);
+            cy.renameSelectedNode(selectors.itemForm, selectors.editItemUrl, childItemName);
+        });
+
+        it('child item inherits parent property and sets value', function () {
+            cy.selectNode(selectors.root, selectors.itemClassForm, className);
+            cy.assignValueToProperty(childItemName, selectors.itemForm, `[data-testid="${newPropertyName}"]`, selectors.treeRenderUrl, selectors.editItemUrl);
+        });
+    });
+
+    describe('Delete property', () => {
+        it('Remove property from main item class', function() {
+            cy.removePropertyFromClass(selectors.root, className, newPropertyName, selectors.itemClassForm, selectors.editClass,  selectors.classOptions, selectors.editClassUrl);
+        });
```

Comment:
I think here we need some check, for example go to item and check that property is removed

Reply:
added 👍